src/tools/Reporting/Reporting/Build.cs

```diff
@@ -25,5 +25,12 @@ public sealed class Build
         public string BuildName { get; set; }
 
         public DateTime TimeStamp { get; set; }
+
+        public Dictionary<string, string> AdditionalData { get; set; } = new Dictionary<string, string>();
```

Comment:
If you decide you should have the method for interacting with this the dictionary should be private.

Reply:
This will stay public since we now switched to manipulating the dictionary from callsite.